.gitignore

```diff
@@ -19,6 +19,7 @@ data/processed/pubmed/*
 !data/processed/pubmed/pubmed-ft-retrieved.rds
 !data/processed/pubmed/pubmed-main.rds
 #data/raw/open-access/
+data/raw/open-access/oa-roadoi.rds
```

Comment:
Why git ignoring this?

Reply:
@maia-sh, it's ~8MB so I think best not to check it in. That's also why I didn't check in timestamped versions of the `trials.csv` and `trials.rds` files. I stored all timestamped files (including the roadoi data) elsewhere. Happy to consider other options!

Reply:
I figured we might want to use the `oa-roadoi.rds` file more regularly, which is why I left it in the repo and ignored it.

Reply:
makes sense!